website/addons/osfstorage/tests/test_views.py

```diff
@@ -553,3 +554,61 @@ def test_cannot_delete_root(self):
         resp = self.delete(self.root_node, expect_errors=True)
 
         assert_equal(resp.status_code, 400)
+
+class TestFileTags(StorageTestCase):
+
+    def test_file_add_tag(self):
+        file = self.node_settings.get_root().append_file('Good Morning.mp3')
+        assert_not_in('Kanye_West', file.tags)
+        url = "/api/v1/project/{pid}/osfstorage/{fid}/tags/{tag}/".format(
```

Comment:
You should be able to use the `api_url_for()` helper instead of writing out the route.